python/rmm/_lib/device_buffer.pyx

```diff
@@ -320,17 +319,53 @@ cpdef DeviceBuffer to_device(const unsigned char[::1] b, uintptr_t stream=0):
     return DeviceBuffer(ptr=p, size=s, stream=stream)
 
 
+@cython.boundscheck(False)
+cdef void copy_async(const void* src,
```

Comment:
Since this is used only internally, we should make its name start with an underscore (Python's way of declaring something internal, hides it from certain autocompletions and whatnot)
```suggestion
cdef void _copy_async(const void* src,
```